### ISSUE TITLE
WCAG usability issue, can't select payment using only a keyboard.

### ISSUE THREAD

**Issue by karmivartio** [paytrail/paytrail-for-woocommerce#213]
paytrail-for-woocommerce payment selection can't be done by all-by keyboard. 

Clicking payment group (_<div class="paytrail-provider-group"_) with mouse is required, because this has only onclick event listener.

Tested Paytrail for WooCommerce 2.5.2 version.

Guideline 2.1 Keyboard Accessible: Make all functionality available from a keyboard. 
https://www.w3.org/TR/WCAG20/#keyboard-operation-keyboard-operable

**Steps to Reproduce**

Try to make payment in Woocommerce with Paytrail using only a keyboard. We can't open payment group using only a keyboard, one mouse click is required.

**Expected behaviour**

Whole order should be possible to done using only a keyboard, so thus payment group should be possible to opened using only a keyboard.

**Actual behaviour**

One mouse click is required to open group, payment method can be selected after this by up,down,left,right keys.

**Additional context**

assets/js/main.js needs also onKeyUp, onKeyDown, or onKeyPress event listener, so that user can select payment group and payment by keyboard.

Don't know which key is commonly used to open this group: tab, enter, or space keys?

https://stackoverflow.com/questions/48575674/how-to-add-a-keyboard-listener-to-my-onclick-handler

![Image](https://github.com/user-attachments/assets/ebd439e0-5125-4bea-b41c-714ca8a6c0c4)

**Comment by kotivuori** [comment]
Thanks for reporting this @karmivartio. We are working on updating the provider view so that it meets the accessibility guidelines.

**Comment by kotivuori** [comment]
This has been resolved in release 2.5.3
